src/types/book.ts

```diff
@@ -17,6 +17,6 @@ export class Book implements BookType {
 
         this.amazonProductURL = `https://www.amazon.com/dp/${this.amazonProductID}?tag=${AMAZON_TRACKING_ID}&language=en_US`;
 
```

Comment:
- Please change `amazonProductURL` to a getter
- Please keep the existing `amazonImageURL` to a getter; we will use this as a backup when the image file does not exist.
- Please add a NEW getter named `imageURL`, and update the source on the component to use this instead